src/Form/GCSettingsForm.php

```diff
@@ -152,6 +152,9 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $settings = $this->config('openy_gated_content.settings');
     $settings->setData($form_state->getValue('app_settings'));
+    $settings->set('event_add_to_calendar', $form_state->getValue('event_add_to_calendar'));
```

Comment:
Please remove these lines, no need to set params separately, they already in `app_settings`